test/gtest/uct/ib/test_rc.cc

```diff
@@ -680,6 +680,12 @@ void test_rc_flow_control::test_pending_grant(int wnd)
     send_am_messages(m_e1, 1, UCS_ERR_NO_RESOURCE);
     EXPECT_LE(get_fc_ptr(m_e1)->fc_wnd, 0);
 
+    if (wait_fc_seq != NULL) {
+        uint64_t fc_seq_value = *wait_fc_seq;
+        wait_for_value(wait_fc_seq, fc_seq_value + 1, true);
+        EXPECT_GT(*wait_fc_seq, fc_seq_value);
+    }
```

Comment:
why we should do it in the end?
my idea to wait for resending while receiver's entity is disabled

Reply:
to remove fc_seq code (which is dc) from test_rc

Reply:
> to remove fc_seq code (which is dc) from test_rc

ic, but unfortunately, it wouldn't work, since remote entity should be disabled until we make sure that `fc_seq` is updated